mappings/net/minecraft/item/Item.mapping

```diff
@@ -213,17 +231,59 @@ CLASS net/minecraft/class_1792 net/minecraft/item/Item
 		FIELD field_8019 maxDamage I
 		FIELD field_8020 maxCount I
 		METHOD method_19265 food (Lnet/minecraft/class_4174;)Lnet/minecraft/class_1792$class_1793;
+			COMMENT When set, any item configured with this Settings instance will be edible based on the provided {@link FoodComponent}.
+			COMMENT
+			COMMENT @return this instance
 			ARG 1 foodComponent
+				COMMENT configured food properties for any item using this Settings instance
 		METHOD method_24359 fireproof ()Lnet/minecraft/class_1792$class_1793;
+			COMMENT If called, any item with this Settings instance is immune to fire and lava damage.
+			COMMENT
+			COMMENT @return this instance
 		METHOD method_7889 maxCount (I)Lnet/minecraft/class_1792$class_1793;
+			COMMENT Sets the maximum stack count of any ItemStack with an Item using this Settings instance.
+			COMMENT
+			COMMENT <p>Note that a count over 64 leads to unreliable behavior in GUIs.
+			COMMENT Damageable items can not have a maximum stack count (they default to 1).
+			COMMENT An Item.Settings' max count defaults to 64.
+			COMMENT
+			COMMENT @throws RuntimeException if this Settings instance also has a max damage value set
+			COMMENT @return this instance
 			ARG 1 maxCount
+				COMMENT maximum stack count of any  ItemStack with an item using this Settings instance
```

Comment:
```suggestion
				COMMENT maximum stack count of any ItemStack with an item using this Settings instance
```

Reply:
Hmm, @Draylar mind consider taking this real quick? We can probably open an issue for assorted javadoc typos in the future.